Archive Auction and Crowdloans

filippoweb3 opened the pull request:
Also search for `bid`, `slot`, etc

Replace with `purchase`, `core`, etc where applicable

Rm Auction Admin OpenGov Track info

Add Coretime Chain to Polkadot Metadata Explorer

Commit:
edits

Commit:
edits

- move to archive
- fix links in archived pages
- rm auction mentions where applicable

Commit:
standardized archives

Commit:
replacements

slot -> core where applicable

Commit:
fix

filippoweb3 commented:
Local build OK, Staging Polkadot and Kusama OK.

Note: no find and replace, all changes made manually.

profradha commented on docs/build/build-guide.md:
```diff
@@ -329,7 +329,6 @@ For a complete list of tools, please take a look here:
 
 ## Resources
 
-- [System (Common Good) Parachains](https://polkadot.network/blog/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/)
```
This us a link to system chains blog. Can stay

filippoweb3 replied:
The link is old (2021), centered around auctions. We have a Wiki page about system chains that we can link to it. Wdyt?

profradha replied:
Sounds good. I'm talking to distractive to get the blog content updated.

profradha commented on docs/build/build-parachains.md:
```diff
@@ -137,7 +137,7 @@ parallelizable objects.
 These could be in the form of:
 
-- System level chains (permanent chains): [leased slots](../learn/learn-auction.md)
+- System level chains (permanent chains): via [agile coretime](../learn/learn-agile-coretime.md)
```
Agile coretime and related purchases do not apply to system chains

```suggestion
- System level chains (permanent chains)
```

profradha commented on docs/build/build-parachains.md:
```diff
@@ -363,24 +363,22 @@ are available.
 
-To include your parachain into the Polkadot network, you will need to acquire a parachain slot.
+To include your parachain into the Polkadot network, you will need to reserve some time on a relay
+chain core.
```
```suggestion
To include your parachain into the Polkadot network, you will need to reserve a core on the relay
chain.
```

profradha commented on docs/build/build-parachains.md:
```diff
@@ -363,24 +363,22 @@ are available.
-deprecated and parachains can [purchase coretime](../learn/learn-guides-coretime-marketplaces.md) to
-produce blocks
-[continuously or on-demand](../learn/learn-parachains.md#parachains-vs-on-demand-parachains).
+Time on a core is acquired through [coretime](../learn/learn-agile-coretime.md) purchased with DOT
```
```suggestion
[Coretime](../learn/learn-agile-coretime.md) can be purchased with DOT
```

profradha commented on docs/build/build-parachains.md:
```diff
@@ -363,24 +363,22 @@ are available.
 ## Resources
 
 - [Parachain tutorials](https://docs.substrate.io/reference/how-to-guides/parachains/)
-- [System (Common Good) Parachains](https://polkadot.network/blog/common-good-parachains-an-introduction-to-governance-allocated-parachain-slots/)
```
This link can stay

profradha commented on docs/general/faq.md:
```diff
@@ -286,18 +286,18 @@ Parachains need collators to continue to progress, so it wouldn't be unreasonabl
 implementers.
 
-### Are parachains ephemeral? What happens when a parachain loses the next auction?
+### Are parachains ephemeral? What happens when a parachain's time on a core expires?
```
```suggestion
### Are parachains ephemeral? What happens when a parachain fails to renew its coretime?
```

profradha commented on docs/general/faq.md:
```diff
@@ -286,18 +286,18 @@ Parachains need collators to continue to progress, so it wouldn't be unreasonabl
-becoming an on-demand parachain instead would be a suitable choice. On-demand parachains are still
-secured by the relay chain, but don't need to hold a parachain slot and can produce a block when its
-economically feasible for them. For more on parachains please see the
+When a parachain's time on a core expires, that parachain has a few options. In most cases, becoming
```
```suggestion
When a parachain's coretime has not been renewed, that parachain has a few options. In most cases, becoming
```

DrW3RK reviewed (commented):
Will review in batches as the PR is big.

filippoweb3 committed:
Update docs/build/build-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/build/build-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/build/build-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/faq.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/faq.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 commented:
Outdated blog posts and videos talking about auctions and/or outdated links:
- https://polkadot.network/blog/the-launch-of-parachains/
- https://medium.com/polkadot-network/parathreads-pay-as-you-go-parachains-7440d23dde06
- https://medium.com/polkadot-network/polkadot-bridges-connecting-the-polkadot-ecosystem-with-external-networks-1118916392e3
- https://www.crowdcast.io/e/polkadot-path-of-a-parachain-block?utm_source=profile&utm_medium=profile_web&utm_campaign=profile (not working)

Commit:
Radha's feedback + rm old links

filippoweb3 commented:
@DrW3RK lmk if you have some time to review this or should I ask a further review by @CrackTheCode016

profradha commented on docs/general/faq.md:
```diff
@@ -286,18 +286,18 @@ Parachains need collators to continue to progress, so it wouldn't be unreasonabl
+When a parachain's coretime has not been renewed, that parachain has a few options. In most cases, becoming
+an on-demand parachain instead would be a suitable choice. On-demand parachains are still secured by
+the relay chain, but don't need to hold a core and can produce a block when its economically
+feasible for them. For more on parachains please see the
```
```suggestion
When a parachain could not renew its bulk coretime and needs to keep its chain live, there are a couple of options to consider. One option is to explore secondary coretime marketplaces on [Lastic](https://www.lastic.xyz/) or [RegionX](https://app.regionx.tech/?network=polkadot). The other option is run as an on-demand parachain, and purchase coretime to produce one block at a time. On-demand parachains are still secured by the relay chain but don't need to hold a core and can produce a block when it's economically feasible for them. For more information, please refer to the
```

profradha commented on docs/general/getting-started.md:
```diff
@@ -120,7 +120,8 @@ OpenGov) to orchestrate decisions, including accessing funds from the
 The [DOT token](../learn/learn-DOT.md) gives you the power to participate in Polkadot OpenGov,
-[staking](../learn/learn-staking.md), and [parachains' crowdloans](../learn/learn-crowdloans.md).
+[staking](../learn/learn-staking.md), and
+[parachains' crowdloans](../learn/archive/learn-crowdloans.md).
```
```suggestion
and [staking](../learn/learn-staking.md).
```

profradha commented on docs/general/glossary.md:
```diff
@@ -169,11 +169,15 @@ The process of a group of entities to agree on a particular data value (such as
 
+## Coretime
+
+Time on a relay chain core than can be purchased via Coretime system chain.
```
Not fully true. On-demand purchases happen on relay chain. 

We already have definition for this here https://wiki.polkadot.network/docs/learn-agile-coretime#coretime 

```suggestion
The time allocated for utilizing a core that can be purchased in bulk or on demand. It is measured in relay chain blocks.
```

profradha commented on docs/general/polkadot-direction.md:
```diff
@@ -49,14 +48,14 @@ Polkadot can be directed towards the following main changes.
-Previously, securing a parachain slot was a competitive process through an
-[auction mechanism](../learn/learn-auction.md). With coretime rental, there is no need for slot
+Previously, securing a core was a competitive process through an
+[auction mechanism](../learn/archive/learn-auction.md). With coretime rental, there is no need for
```
```suggestion
[auction mechanism](../learn/archive/learn-auction.md). With Agile Coretime, there is no need for
```

profradha commented on docs/general/polkadot-v1.md:
```diff
@@ -73,12 +73,12 @@ meaning that transactions can be made between different parachains.
 
-The Polkadot relay-chain also manges [crowdloans](../learn/learn-crowdloans.md),
-[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
+The Polkadot relay-chain also manges [crowdloans](../learn/archive/learn-crowdloans.md),
```
```suggestion
The Polkadot relay-chain managed [crowdloans](../learn/archive/learn-crowdloans.md),
```

profradha commented on docs/general/polkadot-v1.md:
```diff
@@ -73,12 +73,12 @@ meaning that transactions can be made between different parachains.
-and [governance](../learn/learn-polkadot-opengov.md). Parachain slots or cores are leased in 6-month
-chunks for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for
-lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.
+and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores are leased in 6-month chunks
```
```suggestion
and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores were leased in 3-month chunks
```

profradha commented on docs/general/polkadot-v1.md:
```diff
@@ -73,12 +73,12 @@ meaning that transactions can be made between different parachains.
-chunks for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for
-lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.
+and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores are leased in 6-month chunks
+for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for lease
```
```suggestion
for a maximum of two years, and crowdloans allowed users to trustlessly loan funds to teams for lease
```

profradha commented on docs/general/polkadot-v1.md:
```diff
@@ -73,12 +73,12 @@ meaning that transactions can be made between different parachains.
-lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.
+and [governance](../learn/learn-polkadot-opengov.md). Relay chain cores are leased in 6-month chunks
+for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for lease
+deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.
```
```suggestion
deposits in exchange for pre-sale tokens. There was no other way you could access cores on Polkadot 1.0.
```

profradha commented on docs/general/polkadot-v1.md:
```diff
@@ -142,7 +142,7 @@ further improved such that the tasks which are currently managed on the relay ch
 [system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
 the relay-chain to provide shared security and interoperability. Blockspace is only accessible
-through slot auctions, but an auction winner has access to a "freighter of blocks" regardless it is
+through auctions, but an auction winner has access to a "freighter of blocks" regardless it is
```
Looking at the wording in this doc on Polkadot v1, it makes sense to talk about auctions and crowdloans in past tense.

filippoweb3 replied:
will go through that page in more detail and check

profradha commented on docs/learn/learn-DOT.md:
```diff
@@ -88,13 +88,10 @@ required to participate in the network will vary depending on the activity that
 about staking on Polkadot visit [the dedicated staking page](./learn-staking.md).
 
-### DOT for Parachain Slot Acquisition
+### DOT to Access the Relay Chain
```
```suggestion
### DOT to Access Cores on the Relay Chain
```

profradha commented on docs/learn/learn-agile-coretime.md:
```diff
@@ -410,9 +405,9 @@ the end of the region in which its slot would have expired.
-Auctions take place until the launch of coretime. With the runtime upgrade, enabling Agile Coretime,
-any still ongoing auction will be canceled. Existing active leases of previous auctions will be
-migrated to Agile Coretime: In case you have a lease secured, you automatically get corresponding
+Auctions took place until the launch of agile coretime with Polkadot runtime upgrade on the 19th of
```
```suggestion
Auctions took place until the launch of Agile Coretime with Polkadot runtime upgrade on the 19th of
```

profradha commented on docs/learn/learn-inflation.md:
```diff
@@ -84,22 +84,22 @@ network, please see [here](https://research.web3.foundation/Polkadot/overview/to
 
 Briefly, the ideal staking rate can be calculated as follows:
 
-`0.75 - auction_proportion`
```
These are variables from the code. There is no `auction_proportion` variable in the code.

It makes sense to redo this inflation section as things have changed recently

filippoweb3 replied:
agree, I will update to most recent changes

profradha commented on docs/learn/learn-parachains.md:
```diff
@@ -9,13 +9,13 @@ slug: ../learn-parachains
-<MessageBox message="Parachain Slot Auctions will be deprecated right after [Agile Coretime](./learn-agile-coretime.md) is activated on the network. For existing parachains, the remainder of the lease will automatically be converted to coretime. See more information [here](./learn-agile-coretime.md#implementation)." />
+<MessageBox message="[Agile Coretime](./learn-agile-coretime.md) is activated on the network, and parachain slot auctions and crowdloans have been deprecated. For existing parachains, the remainder of the lease is automatically converted to coretime. See more information [here](./learn-agile-coretime.md#implementation). For decentralized, transparent, and regulatory-compliant fundraising within t ...
 
 :::info Testing on Rococo
```
Rococo will be deprecated on Oct 14th. Will add an issue on the Wiki to replace its mentions with Paseo

filippoweb3 replied:
Added, see https://github.com/w3f/polkadot-wiki/issues/6284

profradha commented on docs/learn/learn-parachains.md:
```diff
@@ -106,7 +106,7 @@ Some examples of features you can have on a parachain or parathread:
 for chains considering becoming a [parachain](learn-parachains.md) and joining the network. On a
 high level, shared security means that all parachains that are connected to the relay chain by
-leasing a parachain slot will benefit from the economic security provided by the relay chain
+leasing a core will benefit from the economic security provided by the relay chain
```
```suggestion
accessing a core will benefit from the economic security provided by the relay chain
```

profradha commented on docs/learn/learn-parachains.md:
```diff
@@ -170,28 +170,32 @@ validity.
+can choose to renew it at a fixed cost (only bulk coretime) or buy it on-demand (variable price
+depending on the demand and other market variables) when they need to access the relay chain.
+Parachains without coretime to extend time on a relay chain core will be deprecated to the status of
+a parathread (i.e., a chain with a registered `ParaID` but without access to a core).
```
```suggestion
The DOT (or KSM on Kusama) used to purchase coretime are burned. Before the coretime expires, parachains
can renew it at a fixed cost through a bulk coretime purchase. If the parachain does not purchase bulk coretime, it has an option to purchase coretime on-demand (at a variable price per block,
depending on the demand and other market conditions) when they need to access the relay chain.
Parachains without coretime to extend time on a relay chain core will be deprecated to the status of
a parathread (i.e., a chain with a registered `ParaID` but without access to a core).
```

profradha commented on docs/learn/learn-parachains.md:
```diff
@@ -216,13 +214,12 @@ On-demand parachains (previously called parathreads) are parachains that acquire
+needing to lease a dedicated relay chain core. This is done through economically sharing the scarce
+resource of a core among several competing resources (parachains). Chains that otherwise would not
+be able to acquire a full core or do not find it economically sensible to do so, can participate in
+shared security as the [on-demand coretime](./learn-agile-coretime.md#on-demand-coretime) offers a
```
```suggestion
shared security, as the [on-demand coretime](./learn-agile-coretime.md#on-demand-coretime) offers a
```

profradha commented on docs/learn/learn-system-chains.md:
```diff
@@ -20,8 +20,8 @@ provides.
-than the relay chain. Rather than leasing an execution core by economic means (e.g., auction),
-execution cores are allocated by network [governance](./learn-guides-polkadot-opengov.md).
+than the relay chain. Rather than purchasing time an execution core, execution cores are allocated
+by network [governance](./learn-guides-polkadot-opengov.md).
```
```suggestion
than the relay chain. Rather than purchasing coretime on a marketplace, execution cores for system chains are allocated
through the network [governance](./learn-guides-polkadot-opengov.md).
```

filippoweb3 committed:
Update docs/general/faq.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/getting-started.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/glossary.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-direction.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-v1.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-v1.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-v1.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/general/polkadot-v1.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-DOT.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-agile-coretime.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-parachains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-system-chains.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
fix broken links

Commit:
Radha's feedback

- past tense when talking about auctions in Polkadot 1.0 page

Commit:
Radha's feedback

reverted back to staking rate original text, new staking rate will be addressed in a new PR

Commit:
slots > cores
